Assets/script.js

```diff
@@ -0,0 +1,115 @@
+    }, 
+]
+
+const culture=[
+{
+    "img":"https://img.icons8.com/office/40/000000/maracas.png",
+    "theme":"Cultural MUsic",
+    "tag":"bright future putting the king fast protected by his Kingsmen bright future putting the king fast protected by his Kingsmen"
+},{
+    "img":"https://img.icons8.com/office/40/000000/maracas.png",
+    "theme":"Cultural Dance",
+    "tag":"bright future putting the king fast protected by his Kingsmen bright future putting the king fast protected by his Kingsmen"
+},{
+    "img":"https://img.icons8.com/office/40/000000/mate.png",
+    "theme":"Cultural Food staff",
+    "tag":"bright future putting the king fast protected by his Kingsmen bright future putting the king fast protected by his Kingsmen"
+},{
+    "img":"https://img.icons8.com/office/40/000000/seed-of-life.png",
+    "theme":"Cultural Knowledge",
+    "tag":"bright future putting the king fast protected by his Kingsmen bright future putting the king fast protected by his Kingsmen"
+}
+]
+const menu=document.querySelector('.hamburger-menu');
+const navMenu = document.querySelector('.Navmenu');
+menu.addEventListener('click',()=>{
+    menu.classList.toggle('active');
+    navMenu.classList.toggle('active');
+})
+
+document.querySelectorAll('.nav-links').forEach((navLink) => navLink.addEventListener('click', () => {
+    menu.classList.remove('active');
+    navMenu.classList.remove('active');
+}))
+
+
+window.onload=()=>{
+let speaker=document.createElement('ul')
+console.log(document.querySelector(".speakers"))
+
+speakers.map((item)=>{
+    let ull = document.createElement('li')
+    ull.innerHTML = `
+    <div>
+    <img src="${item.img}" class='speakership' alt="Speaker image">
+    </div>
+    <div class="">
+        <h3>${item.speaker}</h3>
+        <span class='title'>${item.title}</span>
+        <p>${item.tag}</p>
+    </div>
```

Comment:
Kindly, I suggest in this section to use the semantic tag "ARTICLE", this is ideal for repetitive content

Reply:
Kindly, I suggest in the ALT attribute to use the specific content of each image, for example the name of each speaker

Reply:
Kindly, I suggest you remove the CLASS attribute if you don't need it, this will make your code look cleaner and tidier.